gogs_client/entities.py

```diff
@@ -242,14 +243,14 @@ def pull(self):
             return self._pull
 
 
-class TokenInfo(object):
+class TokenInfo(Token):
```

Comment:
Could we just add a new property (`name`) to the existing `Token` class, instead of introducing a new subclass? I think it would be best to keep the class hierarchies simple, and don't see a pressing need for a subclass here.